src/prop-converter.js

```diff
@@ -29,18 +33,41 @@ import {
   disablePreset,
 } from './enter-leave-presets';
 import { isElementAnSFC, omit } from './helpers';
+import type {
+  Animation,
+  AnimationProp,
+  Presets,
+  FlipMoveProps,
+  FlipMoveDefaultProps,
+  ConvertedProps,
+  DelegatedProps,
+} from './typings';
+
+const nodeEnv: string = process && process.env && process.env.NODE_ENV
+  ? process.env.NODE_ENV
+  : 'development';
```

Comment:
Hm, I think we should just tell Flow not to worry about it with `$FlowFixMe` then. 

The problem with this solution is that webpack's `definePlugin` will transform `process.env.NODE_ENV`. So it'll transform into this:

```js
const nodeEnv: string = process && process.env && 'production'
  ? 'production'
  : 'development';
```

Which will erroneously resolve to 'development'.